Assets/MRTK/Core/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -19,5 +19,12 @@ public interface IMixedRealityCameraSystem : IMixedRealityEventSystem, IMixedRea
         /// Is the current camera displaying on an opaque (VR / immersive) or a transparent (AR) device
         /// </summary>
         bool IsOpaque { get; }
+
+        /// <summary>
+        /// Override the camera's projection matrices for a smaller field of view
+        /// but rendered content will have more detail.  If holograms are not stable,
+        /// change the Stereo Rendering Mode from "Single Pass Instanced" to "Multi Pass."
+        /// </summary>
+        bool ProjectionOverrideEnabled { get; set; }
```

Comment:
Because we're adding a new property to an existing interface, this PR should also contain a change to the ReleaseNotes.md about the breaking change.... but this is probably gonna be tricky right now because we're still actively merging 2.5 stabilization notes into mrtk_development (i.e. this will just cause merge conflicts that keep happening).

https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8684

Reply:
This feels like a fairly specific property that we might want to add a new interface for instead of adding to the existing one. Not all devices support this concept, so not all camera systems would need to either.